api/BaseAudioContext.json

```diff
@@ -517,11 +517,11 @@
             "description": "Implemented on <code>AudioContext</code>",
             "support": {
               "chrome": {
-                "version_added": "10",
+                "version_added": "56",
```

Comment:
Its parent feature, `createConstantSource`.  As the lint check this concerns states, _a subfeature cannot be supported by a browser if its parent doesn’t support it._  `createConstantSource` doesn’t have Chrome support until version 56, so how can its subeature, the one currently in question, be supported in an earlier version?

Reply:
Because this member was moved from another interface, and that other interface inherits from BaseAudioContext.

Reply:
Well...that complicates things, to say the least.  If it was moved over though, wouldn’t that mean that this was _really_ implemented in this API in the version when the migration occurred?  Most of the time, a feature implementation that’s said to be before its parent would be a mistake, which is what the linter checks against.

Reply:
Not in this case because it's under a node named `implemented_on_audio_context`, referring to the fact that this was implemented on AudioContext from version 10 onwards.

Reply:
So this will be a common scenario, where an `implemented_on_*` does not apply for the version number rule?

Reply:
This is quite a special case, as there's only two `implemented_on_*` subfeatures in the entire data, one that already doesn't raise issues in the linter, and then this one.  It feels like we'd just be adding a special case into the linter that really only affects this one file (which isn't hard to implement, but for one file out of thousands...).

Another opinion may help in this case.  @Elchi3 @ddbeck, what do you think?

Reply:
My experience with these kinds of tools is that some things should only be warnings and sometimes those warnings will hit edge cases that just need to be ignored.